projects/golang/go/Makefile

```diff
@@ -20,19 +20,32 @@ ifeq ($(ARCHITECTURE), ARM64)
 endif
 ARCH_LOWER=$(shell echo $(ARCHITECTURE) | awk '{print tolower($0)}')
 
-
 ARCH_RPM_OUT_PATH?=golang-$(GIT_TAG)/releases/$(BUILD_ID)/RPMS/$(RPM_ARCH)
 NOARCH_RPM_OUT_PATH?=golang-$(GIT_TAG)/releases/$(BUILD_ID)/RPMS/noarch
 ARCHIVES_OUT_PATH?=golang-$(GIT_TAG)/releases/$(BUILD_ID)/archives
 
+AWS_ACCOUNT_ID?=$(shell aws sts get-caller-identity --query Account --output text)
```

Comment:
yep, exactly -- so that the account ID will be set to the user who is making it

so when we construct 'IMAGE_REPO' [on 30 ](https://github.com/aws/eks-distro-build-tooling/pull/667/files/5247bec170ca07546ee98ff145561a8dd8fe9e2a#diff-5f1e44500fb103dc0ce96c63ccc7dca3b6dbceb62e74976286539988dd20f4a6R30) it'll be the ECR in the callers account